### PR TITLE
Update `method` to `name`

### PR DESCRIPTION
**Pull request by shawntabrizi**
When working through some of the event/error handling, I found that the field `method` was actually `name`:

image

If you search `{method}` elsewhere in the docs, there are a few more instances, but I wasn't confident enough to update all of them. This should be a good start though.

### CODE REVIEW

**Comment by jacogr** [comment]
See https://github.com/polkadot-js/api/blob/master/packages/types/src/generic/Event.ts#L122-L127

The `method` getter is exposed. Where it is correct is in the second change, the documentation as returned from the meta call refers to `name`, as per the metadata.

File: docs/api/cookbook/tx.md [hunk 1]
```diff
@@ -71,16 +71,16 @@ api.tx.balances
       if (dispatchError.isModule) {
         // for module errors, we have the section indexed, lookup
         const decoded = api.registry.findMetaError(dispatchError.asModule);
-        const { documentation, method, section } = decoded;
+        const { documentation, name, section } = decoded;
```
Comment (jacogr) [inline docs/api/cookbook/tx.md:74]: +1, as per metadata

File: docs/api/examples/promise/transfer-events.md [hunk 2]
```diff
@@ -48,8 +48,8 @@ async function main () {
         console.log('Included at block hash', status.asInBlock.toHex());
         console.log('Events:');
 
-        events.forEach(({ event: { data, method, section }, phase }) => {
-          console.log('\t', phase.toString(), `: ${section}.${method}`, data.toString());
+        events.forEach(({ event: { data, name, section }, phase }) => {
```
Comment (jacogr) [inline docs/api/examples/promise/transfer-events.md:51]: This is called `method` (as per the inked source). Same with the remaining 2 changes.

**Comment by shawntabrizi** [comment]
@jacogr should be fixed, sorry :P

**Comment by jacogr** [comment]
No. Actually realized after this that error is not consistent. Additionally in the metadata it doesn’t quite match.

**Review by jacogr — APPROVED** [review]
Thanks a million!

**Pull request merged**